### ISSUE TITLE
[chart/datadog] >=2.27.10 fails to deploy with: unknown capability: "CAP_SYS_CHOWN"

### ISSUE THREAD

**Issue by iAnomaly** [DataDog/helm-charts#484]
**Describe what happened:**
`system-probe` container of the `datadog` (agent) Pod fails to start with chart version >=`2.27.10` with this Deployment/DaemonSet controller event error:
```
│   Warning  Failed     5m35s                 kubelet            Error: Error response from daemon: invalid CapAdd: unknown capability: "CAP_SYS_CHOWN"    
```
It looks like this functionality was just added to the chart values.yaml in #483. A workaround for version >=`2.27.10` can be used by overriding the input values with the old values:
```
agents:
  containers:
      systemProbe:
        securityContext:
          capabilities:
            add: ["SYS_ADMIN", "SYS_RESOURCE", "SYS_PTRACE", "NET_ADMIN", "NET_BROADCAST", "NET_RAW", "IPC_LOCK"]
```

**Describe what you expected:**
All Pod containers to start successfully.

**Steps to reproduce the issue:**
Deploy chart version >=`2.27.10` 

**Additional environment details (Operating System, Cloud provider, etc):**
Kubernets 1.21 (EKS)

**Comment by clamoriniere** [comment]
Hi @iAnomaly 

Sorry for the issue. 
the PR #486 should fix the issue

**Comment by iAnomaly** [comment]
Confirmed fixed by #486. Thanks @clamoriniere.
